canonical-web-and-design/maas.io
# Resolve js map errors

## Done
- Copy over all the build assets from the 3rd party packages including the map files
- Only load discourse-red module on docs pages

## QA
- Load the homepage of the demo
- Check the console no longer warns of missing JS map files
- Go to docs and check it loads the discourse RAD module
